deps/wazuh_testing/wazuh_testing/tools/monitoring.py

```diff
@@ -779,8 +779,8 @@ def run(self, *args):
 
         # Give proper permissions to socket
         if isinstance(self.listener_socket_address, str):
-            uid = pwd.getpwnam('wazuh').pw_uid
-            gid = grp.getgrnam('wazuh').gr_gid
+            uid = pwd.getpwnam('ossec').pw_uid
+            gid = grp.getgrnam('ossec').gr_gid
```

Comment:
Revert this change

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1345/commits/ba4f4aa7304d2cf27ce9776b834e87e1cbeed8c4